scripts/irods/test/resource_suite.py

```diff
@@ -513,68 +513,80 @@ def test_local_iput_onto_specific_resource(self):
         if os.path.exists(datafilename):
             os.unlink(datafilename)
 
-    @unittest.skipIf(True, 'Enable once #2634 is resolved')
     def test_local_iput_interrupt_directory(self):
-        # local setup
-        datadir = "newdatadir"
-        os.makedirs(datadir)
+        local_dir_path = os.path.join(self.admin.local_session_dir, "newdatadir")
+        collection_path = os.path.join(self.admin.session_collection, 'newdatadir')
         datafiles = ["file1", "file2", "file3", "file4", "file5", "file6", "file7"]
+        restartfile = os.path.join(self.admin.local_session_dir, 'collectionrestartfile')
+        iputcmd = f'iput -X {restartfile} -r {local_dir_path} {collection_path}'
```

Comment:
Could `iputcmd` be a list? Asking since I see it's used in `icommand` stuffs.

Reply:
Done

Reply:
Done